installer/py3.10-linux-x86_64-cuda-reqs.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.10
 # To update, run:
 #
-#    pip-compile --allow-unsafe --generate-hashes --output-file=installer/py3.10-linux-x86_64-cuda-reqs.txt requirements.in
+#    pip-compile --allow-unsafe --generate-hashes --output-file=installer/py3.10-windows-x86_64-cuda-reqs.txt installer/requirements.in
```

Comment:
"windows"?

Reply:
grr...

Reply:
maybe it doesn't matter, just smells funny

Reply:
fixed